frontend/src/pages/devices/List.tsx

```diff
@@ -223,16 +223,17 @@ export const List = () => {
                         // Once we have one warning, we'll only add additional ones for `Adding` codes.
                         continue
                     }
-                    if (lc.wasEnabledAt) {
-                        const wc = Date.parse(lc.wasEnabledAt)
-                        const minutesBetween = (wc - sa) / 1000 / 60
-                        if (expectedResponseInMinutes < minutesBetween) {
-                            const distance = formatDistance(sa, wc)
-                            warnings.push(<>Slow to Respond (took { distance } to add code { lc.code })</>)
-                        }
+                    if (lc.note === 'Lock code present.'){
```

Comment:
If this was our only lock code:

```json
    {
      "code": "1234",
      "endAt": "2030-01-01T01:00:00Z",
      "id": "a25cd2cd-bf1b-4193-9c71-1d796d47aae7",
      "note": "Lock code present.",
      "reservation": {
        "id": "",
        "sync": false
      },
      "status": "Enabled",
      "startAt": "2020-01-01T01:00:00Z",
      "startedAddingAt": "2020-01-01T01:00:00Z",
      "wasEnabledAt":     "2020-01-02T01:00:00Z",
      "startedRemovingAt": null,
      "wasCompletedAt": null
    }
```

I think we'd want to show an error saying that the lock took 24 hours to add (it's currently present, but the lock may not be very responsive).